modules/apps/dynamic-data-mapping/dynamic-data-mapping-web/src/main/java/com/liferay/dynamic/data/mapping/web/internal/exportimport/data/handler/DDMStructureStagedModelDataHandler.java

```diff
@@ -440,8 +440,8 @@ else if (_isModifiedStructure(existingStructure, structure)) {
 				structure.getType(), serviceContext);
 		}
 
-		_importDEDataDefinitionFieldLinks(
-			importedStructure, portletDataContext, structure);
```

Comment:
@carolmariaabb I think we need to import all structures first, because the DEDataDefinitionLink could be referencing a different structure (through its classPK field) than the structure it is tied to through its ddmStructureId field

Reply:
No problem. Thanks for explaining!